Feature/11210 Change uptime to uptimerobot (#182)
renuo/applications-setup-guide: ruby_on_rails/uptimerobot.md

# Uptimerobot Monitoring

To ensure that our application is always up and running, we offer a monitoring
service to the customers.

When we are still developing a new application, the uptimerobot check should not be
setup to avoid premature costs. Once the go-live date is very close, we enable
the monitoring only for the `master` environment, which *must* have a paid
dyno.

## Setup

You will need Renuo-CLI to be set up and at the newest version:
`gem install renuo-cli` --> see [renuo-cli](https://github.com/renuo/renuo-cli)

1. Run the command `renuo setup-uptimerobot [url] [scope]`

>    * Where `url` is the address that you want to setup monitoring for

Reviewer: `https://[project-name]-master.renuoapp.ch/home/check` or the `https://customdomain/home/check`

>    * Where `url` is the address that you want to setup monitoring for

Reviewer: ```suggestion
   * Where `url` is the address you want to monitor. e.g. `https://[project-name]-master.renuoapp.ch/home/check` or `https://customdomain/home/check`
```
